tests/atomic_ref_stress/atomic_ref_stress_common.h

```diff
@@ -172,15 +172,15 @@ class aquire_release {
           refA.store(0);
           refB.store(0);
           sycl::group_barrier(item.get_group());
-          if (item.get_local_id() == 0) {
+          if (item.get_local_id() == sycl::id(0)) {
```

Comment:
I do not think so, the constructors are implicit https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#id-class

Reply:
Same here https://github.com/triSYCL/triSYCL/blob/master/include/triSYCL/detail/small_array.hpp#L381

Reply:
So either the special-casing needs to be in the spec, or this explicit construction is correct.